hw/top_earlgrey/ip/xbar_main/dv/autogen/xbar_cover.cfg

```diff
@@ -0,0 +1,84 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// xbar_cover.cfg generated by `tlgen.py` tool
+
++tree tb.dut
+-module pins_if     // DV construct.
+-module clk_rst_if  // DV construct.
+
+// due to VCS tool issue, can't move this part into begin...end (tgl) or after
```

Comment:
This is fixed in VCS version 2020.12.
https://solvnetplus.synopsys.com/s/detail/a0f3g000000U5cD